File: app/src/main/java/edu/vanderbilt/vandyvans/models/ArrivalTime.java
package edu.vanderbilt.vandyvans.models;

/**
 * Created by athran on 3/15/14.
 */
public class ArrivalTime {

    public final Stop  stop;
    public final Route route;
    public final int   minutes;

    public ArrivalTime(Stop  _stop,
                       Route _route,
                       int   _min) {
        stop    = _stop;
        route   = _route;
        minutes = _min;
    }

    @Override
    public String toString() {
        return new StringBuilder("ArrivalTime(")
                .append(stop.name)
                .append(", ")
                .append(route.name)
                .append(", ")
                .append(minutes)
                .append(")")
                .toString();
    }

}

File: model/src/main/scala/com/vandyapps/vandyvans/models/ArrivalTime.scala
package com.vandyapps.vandyvans.models

/** The prediction of when a van of a particular route will arrive at a stop.
  *
  * @param stop The stop that the arrival time pertains to
  * @param route The route
  * @param minutes how many more minutes before a van arrives to the stop.
  */
case class ArrivalTime(stop: Stop, route: Route, minutes: Int)

